lib/minfraud/components/payment.rb

```diff
@@ -5,18 +5,128 @@ class Payment < Base
-        :sagepay, :securetrading, :simplify_commerce, :skrill, :smartcoin, :solidtrust_pay, :sps_decidir, :stripe, 
-        :telerecargas, :towah, :transact_pro, :usa_epay, :vantiv, :verepay, :vericheck, :vindicia, :virtual_card_services, 
-        :vme, :vpos, :wirecard, :worldpay
+        :adyen,
+        :affirm,
+        :afterpay,
+        :altapay,
+        :amazon_payments,
+        :american_express_payment_gateway,
+        :authorizenet,
+        :balanced,
+        :beanstream,
+        :bluepay,
+        :bluesnap,
+        :bpoint,
+        :braintree,
+        :ccavenue,
+        :ccnow,
+        :cetelem,
+        :chase_paymentech,
+        :checkout_com,
+        :cielo,
+        :collector,
+        :commdoo,
+        :compropago,
+        :concept_payments,
+        :conekta,
+        :ct_payments,
+        :cuentadigital,
+        :curopayments,
+        :cybersource,
+        :dalenys,
+        :dalpay,
+        :datacash,
+        :dibs,
+        :digital_river,
+        :dotpay,
+        :ebs,
+        :ecomm365,
+        :ecommpay,
+        :elavon,
+        :emerchantpay,
+        :epay,
+        :eprocessing_network,
+        :eway,
+        :exact,
+        :first_data,
+        :g2a_pay,
+        :global_payments,
+        :gocardless,
+        :heartland,
+        :hipay,
+        :ingenico,
+        :interac,
+        :internetsecure,
+        :intuit_quickbooks_payments,
+        :iugu,
+        :klarna,
+        :lemon_way,
+        :mastercard_payment_gateway,
+        :mercadopago,
+        :mercanet,
+        :merchant_esolutions,
+        :mirjeh,
+        :mollie,
+        :moneris_solutions,
+        :nmi,
+        :oceanpayment,
+        :oney,
+        :openpaymx,
+        :optimal_payments,
+        :orangepay,
+        :other,
+        :pacnet_services,
+        :payeezy,
+        :payfast,
+        :paygate,
+        :paylike,
+        :payment_express,
+        :paymentwall,
+        :payone,
+        :paypal,
+        :payplus,
+        :paysafecard,
+        :paystation,
+        :paytrace,
+        :paytrail,
+        :payture,
+        :payu,
+        :payulatam,
+        :payway,
+        :payza,
+        :pinpayments,
+        :posconnect,
+        :princeton_payment_solutions,
+        :psigate,
+        :qiwi,
+        :quickpay,
+        :raberil,
+        :rede,
+        :redpagos,
+        :rewardspay,
+        :sagepay,
+        :securetrading,
+        :simplify_commerce,
+        :skrill,
+        :smartcoin,
+        :smartdebit,
+        :solidtrust_pay,
+        :sps_decidir,
+        :stripe,
+        :synapsefi,
+        :telerecargas,
+        :towah,
+        :transact_pro,
+        :usa_epay,
+        :vantiv,
+        :verepay,
+        :vericheck,
+        :vindicia,
+        :virtual_card_services,
+        :vme,
+        :vpos,
+        :wirecard,
+        :worldpay,
```

Comment:
https://github.com/rubocop-hq/ruby-style-guide#no-trailing-array-commas

Reply:
I removed the comma. Thank you for the review!